crates/bevy_ui/src/node_bundles.rs

```diff
@@ -105,6 +108,51 @@ pub struct ImageBundle {
     pub z_index: ZIndex,
 }
 
+/// A UI node that is a texture atlas sprite
+#[derive(Bundle, Debug, Default)]
+pub struct AtlasImageBundle {
+    /// Describes the logical size of the node
+    ///
+    /// This field is automatically managed by the UI layout system.
+    /// To alter the position of the `NodeBundle`, use the properties of the [`Style`] component.
+    pub node: Node,
+    /// Styles which control the layout (size and position) of the node and it's children
+    /// In some cases these styles also affect how the node drawn/painted.
+    pub style: Style,
+    /// The calculated size based on the given image
+    pub calculated_size: ContentSize,
+    /// The background color, which serves as a "fill" for this node
+    ///
+    /// Combines with `UiImage` to tint the provided image.
+    pub background_color: BackgroundColor,
+    /// A handle to the texture atlas to use for this Ui Node
+    pub texture_atlas: Handle<TextureAtlas>,
+    /// The descriptor for which sprite to use from the given texture atlas
```

Comment:
```suggestion
    /// The descriptor used to determine which sprite to use from the given texture atlas
```

Reply:
I'll respond to all the docs nits here:  Most of these docs are copied from the existing image bundle, texture atlas bundles, or existing examples.  Arguably that is bad practice but it matches what the surrounding bundles did.  I think for consistency if we change the wording on any of these I would also want to change it in the original spots but that seems like a different PR